packages/shared/locales/en.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
I think the application is not going to allow multi-language. Wouldn't it be better to modify the texts, directly where they are going to be placed?
Instead of modifying the English json

Reply:
Yesterday, I opened an [issue](https://github.com/iotaledger/legacy-migration-tool/issues/24) to remove `i18n` and everything related to it, but I talked to Bego on Slack, and we decided it's better to leave it as it is, just in case we want multilingual support in the future... :pray: